Shunt fx_interpreter graphmodule print on error into tlparse

wconstab opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #158481
* __->__ #158469

Include both the error stacktrace and the graphmodule in a new
structured trace artifact.  Log the shortened version to the console,
and also log a hint to look at the tlparse for more.

cc @ezyang @SherlockNoMad @EikanWang @jgong5 @wenzhe-nrv

wconstab committed:
Shunt fx_interpreter graphmodule print on error into tlparse

Include both the error stacktrace and the graphmodule in a new
structured trace artifact.  Log the shortened version to the console,
and also log a hint to look at the tlparse for more.

[ghstack-poisoned]

ezyang reviewed (approved):
I'd probably also check with whoever added this in the first place. Also, the error message is totally not going to help anyone who doesn't already know how to do this. @williamwen42 this is a perfect use case for the website BTW.

wconstab commented:
> I'd probably also check with whoever added this in the first place

Looks like it was @bdhirsh -  well, i'll go ahead and land it and when he's back we can confirm and change if needed.

> Also, the error message is totally not going to help anyone who doesn't already know how to do this.

I could link to a doc. Would this be the best link for docs? it at least shows usage. i am not sure if we have an official doc page or tutorial that shows this. 
https://github.com/pytorch/tlparse?tab=readme-ov-file#tlparse-parse-structured-pt2-logs

wconstab committed:
Update on "Shunt fx_interpreter graphmodule print on error into tlparse"


Include both the error stacktrace and the graphmodule in a new
structured trace artifact.  Log the shortened version to the console,
and also log a hint to look at the tlparse for more.

cc ezyang SherlockNoMad EikanWang jgong5 wenzhe-nrv

[ghstack-poisoned]
